R/funs.R

```diff
@@ -17,7 +17,17 @@
 #' Use these constants as the \code{position} argument in the \code{\link{create_overlay}} function.
 #'
 #' @export
-overlay_shape_position <- c("top-left", "top-right", "top-center", "bottom-left", "bottom-right", "bottom-center", "middle-left", "middle-right")
+overlay_shape_position <-
```

Comment:
**suggestion:** I have noticed an rendering issue in the documentation with the list of `overlay_shape_position` and for `overlay_edg_position`. The issue was already there.
We may fix it as part of this PR to further improve the documentation.

![image](https://github.com/process-analytics/bpmn-visualization-R/assets/27200110/41983737-c679-44ba-bfbf-1c02283e772e)
_`overlay_shape_position` rendering issue_

Reply:
### After fix

![](https://github.com/process-analytics/bpmn-visualization-R/assets/4921914/5d865663-400f-4a69-84a6-22e3d523a1a8)

![](https://github.com/process-analytics/bpmn-visualization-R/assets/4921914/6ba8aceb-cf04-4371-8cf3-8ed4d1f26301)

Reply:
Nice!